api/test/integration/import-data/xlsx-uploads/sourcing-data-import.spec.ts

```diff
@@ -58,7 +58,7 @@ jest.mock('config', () => {
   return config;
 });
 
-describe('Sourcing Data import', () => {
+describe.skip('Sourcing Data import', () => {
```

Comment:
Skipped in this PR. Sourcing Data Import tests updated in: https://github.com/Vizzuality/landgriffon/pull/204

Reply:
Add a comment with a @TODO explaining why we are skipping it, so we know when we can restore it